src/components/search-modules/Location/Location.jsx

```diff
@@ -45,10 +45,11 @@ const Location = ({ handleUpdate }) => {
 		hospital,
 		vaOnly,
 	} = useSelector((store) => store.form);
+	const defaultCountryValue = country == 'United States';
```

Comment:
Use `===` instead of `==`

For reference
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

Reply:
oops must have missed an extra =, adding now